server/BUILD

```diff
@@ -160,7 +160,35 @@ linux_packages(
         "//server:services/cassandra/logback.xml": "services/cassandra/logback.xml",
         "//server:services/grakn/grakn-core-ascii.txt": "services/grakn/grakn-core-ascii.txt"
     },
-    modes = {
+    permissions = {
+        "services/cassandra/cassandra.yaml": "0777",
+        "./var/lib/grakn/db/queue": "0777",
+        "./var/lib/grakn/db/cassandra": "0777",
+    },
+    symlinks = {
+        "./opt/grakn/core/server/db/": "/var/lib/grakn/db/",
+    },
+)
+
+
+rpm_package(
+    name = "deploy-rpm",
+    package_name = "grakn-core-server",
+    installation_dir = "/opt/grakn/core/server/",
+    version_file = "//:VERSION",
+    spec_file = "//dependencies/distribution/rpm:grakn-core-server.spec",
+    target = ":server-binary",
+    empty_dirs = [
+        "./opt/grakn/core/server/services/lib/",
+        "./var/lib/grakn/db/queue",
+        "./var/lib/grakn/db/cassandra"
+     ],
+    files = {
+        "//server:services/cassandra/cassandra.yaml": "services/cassandra/cassandra.yaml",
+        "//server:services/cassandra/logback.xml": "services/cassandra/logback.xml",
+        "//server:services/grakn/grakn-core-ascii.txt": "services/grakn/grakn-core-ascii.txt"
+    },
+    permissions = {
```

Comment:
same issue, remove `./` in `permissions`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c